content/_data/changelogs/lts.yml

```
@@ -7612,6 +7612,63 @@
     message: |-
       Make previous boot attempt timestamps available to boot-failure.groovy startup hooks (regression in 2.308).
 
+
+- version: "2.346.3"
+  date: 2022-08-10
+  changes:
+
+  - type: rfe
+    category: rfe
+    pull: 6565
+    authors:
+     - dependabot[bot]
+    pr_title: Upgrade Spring Framework from 5.3.19 to 5.3.20
+    references:
+      - pull: 6565
+      - url: https://github.com/spring-projects/spring-framework/releases/tag/v5.3.20
+        title: Spring Framework 5.3.20 changelog
+      - url: https://tanzu.vmware.com/security/cve-2022-22970
+        title: CVE-2022-22970
+      - url: https://tanzu.vmware.com/security/cve-2022-22971
+        title: CVE-2022-22971
+    message: |-
+      Upgrading Spring Framework from 5.3.19 to 5.3.20.
+      Spring Framework 5.3.20 includes 14 fixes and improvements.
+      Additionally, this releases includes fixes for 2 vulnerabilities.
```

Usually the details of changes not specific to Jenkins users are removed from the changelog.
```suggestion
      Spring Framework 5.3.20 includes 14 fixes and improvements.
      Additionally, this releases includes fixes for 2 vulnerabilities.
```